packages/multiline-text-input/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@ui-kit/multiline-text-input",
```

Comment:
Nit but I bet this namespace will be taken. Also nested namespaces don't work. So `@ct-fe/uikit/text-input` won't do. I guess `@ct-fe/uikit-textinput` could be something.

Reply:
yeah good point.

`@commercetools-frontend/ui-kit-text-input` should work?

Reply:
Yes, I think so. Or a completley new scope `#commercetools-uikit/text-input`.